client/components/follow-button/style.scss

```diff
@@ -1,3 +1,5 @@
+$follow-button-gray-disabled: lighten( $gray, 20% );
```

Comment:
It was just to avoid duplication really @jancavan - didn't want to repeat the `lighten` five times when we want the same outcome each time. Are there any Calypso guidelines about this? (Had a peek but couldn't see anything in https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md).